specification/network/resource-manager/Microsoft.Network/stable/2019-08-01/virtualWan.json

```diff
@@ -857,7 +857,313 @@
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "VpnServerConfigurationParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "./network.json#/definitions/TagsObject"
+            },
+            "description": "Parameters supplied to update VpnServerConfiguration tags."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. Returns the details of the VpnServerConfiguration updated.",
+            "schema": {
+              "$ref": "#/definitions/VpnServerConfiguration"
+            }
+          },
+          "201": {
+            "description": "Request received successfully. Returns the details of the VpnServerConfiguration updated.",
+            "schema": {
+              "$ref": "#/definitions/VpnServerConfiguration"
+            }
+          },
+          "default": {
+            "description": "Error.",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "delete": {
+        "operationId": "VpnServerConfigurations_Delete",
+        "x-ms-examples": {
+          "VpnServerConfigurationDelete": {
+            "$ref": "./examples/VpnServerConfigurationDelete.json"
+          }
+        },
+        "description": "Deletes a VpnServerConfiguration.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource group name of the VpnServerConfiguration."
+          },
+          {
+            "name": "vpnServerConfigurationName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the VpnServerConfiguration being deleted."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. VpnServerConfiguration deleted."
+          },
+          "202": {
+            "description": "Request received successfully. VpnServerConfiguration deletion is in progress."
+          },
+          "204": {
+            "description": "No VpnServerConfigurations exist by the name provided."
+          },
+          "default": {
+            "description": "Error.",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
```

Comment:
>"final-state-via": "location" [](start = 10, length = 29)

So PUT is using azure-async-operation and DELETE is using location. Should these be consistent?

Reply:
Yes it was intentional and its similar to other networking resources.